src/Arcus.Messaging.Pumps.ServiceBus/Extensions/MessageCorrelationInfoExtensions.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using Arcus.Messaging.Abstractions;
+using GuardNet;
+using Microsoft.Azure.ServiceBus;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+
+namespace Arcus.Messaging.Pumps.ServiceBus.Extensions
```

Comment:
I tend to move this to the namespace of the type that is being extended to improve discovery, thoughts?

Reply:
Yes, agreed!